pkg/client/src/app/pages/controls/business-services/business-services.tsx

```diff
@@ -164,6 +170,8 @@ export const BusinessServices: React.FC = () => {
         {
           title: (
             <AppTableActionButtons
+              isDeleteEnabled={isAssignedToApplication}
+              tooltipMessage="Remove associated application(s)"
```

Comment:
Shouldn't the tooltip message be or equivalent of "Cannot remove a business service associated with application(s)" ?

Reply:
Updated